x-pack/plugins/security_solution/public/flyout/right/components/analyzer_tree.tsx

```diff
@@ -83,42 +72,24 @@ export const AnalyzerTree: React.FC<AnalyzerTreeProps> = ({
     });
   }, [eventId, openLeftPanel, indexName, scopeId]);
 
-  if (loading) {
-    return <EuiLoadingSpinner data-test-subj={ANALYZER_TREE_LOADING_TEST_ID} />;
-  }
-
-  if (error) {
-    return (
-      <EuiEmptyPrompt
-        iconType="error"
-        color="danger"
-        title={<h2>{ERROR_TITLE(ANALYZER_PREVIEW_TEXT)}</h2>}
-        body={<p>{ERROR_MESSAGE(ANALYZER_PREVIEW_TEXT)}</p>}
-        data-test-subj={ANALYZER_TREE_ERROR_TEST_ID}
-      />
-    );
-  }
-
```

Comment:
the new `ExpandablePanel` component handles loading and error states

Reply:
At first glance it was a little confusing, because analyzer tree is an inner component within analyzer preview, but all the test ids reference analyzer preview (outer component). Since `ExpandablePanel` handles error handling and loading (which is awesome btw!), it's safe to merge analyzer tree into analyzer preview.

Reply:
you are right. I gave it a shot really quickly but I think I'd want to spend a bit more time and to it properly. Let's do it in a follow up PR ([ticket](https://github.com/elastic/security-team/issues/7315) created)